src/Sylius/Bundle/PayumBundle/Storage/DoctrineStorage.php

```diff
@@ -27,18 +28,28 @@
  */
 class DoctrineStorage extends AbstractStorage
 {
-    public function __construct(protected ObjectManager $objectManager, $modelClass)
+    /** @var class-string */
+    protected $modelClass;
+
+    /**
+     * @param class-string $modelClass
+     */
+    public function __construct(protected ObjectManager $objectManager, string $modelClass)
```

Comment:
Won't this work this way?
```suggestion
    public function __construct(protected ObjectManager $objectManager, protected string $modelClass)
```